### ISSUE TITLE
Default Share groups: should it be sitewide and not just the user?

### ISSUE THREAD

**Issue by Theodlz** [skyportal/skyportal#4587]
Please fill out the relevant sections below; remove those that are unused.

**Feature Summary**
Most users of Fritz seem to complain that when uploading any data product without specifying groups to share the data with, the default is to only share to the current user, and not to either all the groups they have access to or sitewide. They assume that it does already, so they get confused.

**Alternative Solutions**
Should we consider making this configurable? Just like we can configure in `config.yaml` what permissions are given to new users, we could maybe configure what groups are used to share data when none are specified (which would be of course not used when groups to share the data with are specified).

**Implementation details**
It could be set in the `config.yaml` configuration file, grabbed in the handlers where needed, and maybe also sent to the frontend using the config handler to set the default groups there as well (though if we don't the handler will still take care of it, but it might be better to avoid confusion the other way around).

**Additional context**
I know the whole data access part of Fritz is a requirement and is important, but we should really consider defaulting to sharing data and not sharing only when specified, rather than the opposite (which is what we are doing now).

**Comment by Theodlz** [comment]
@profjsb @mcoughlin you're input would be required here before I start implementing anything.

**Comment by mcoughlin** [comment]
@Theodlz I think it does make sense to share broadly. But one path may also be to make it more clear in the frontend what happens on upload, I.e explicitly call out share just with you or share with everyone.

**Comment by Theodlz** [comment]
@mcoughlin I agree. Regardless of what we change (or do not change), we should make that clearer.

However this change in behavior wouldn't just be for the frontend, but also the API (or we could consider implementing it for only one of them, that's what I'd like to discuss with the 2 of you). 

Again, this is really something that pretty much everyone using Fritz has brought up at some point. For instance today, we realized some users were asking other users if there was no spectra data yet on multiple sources, when in reality there was for a while already, but the people who reduced it didn't share with any group specifically, so it ended up not being shared with anyone.

**Comment by Theodlz** [comment]
Anyway, I think the data access restriction should be a feature people that don't want to share their data to everyone else need to use, and not the default behavior.

**Comment by robertdstein** [comment]
I would strongly support that the default behaviour, at least in ZTF/Fritz instance, is 'share sitewide'
